pkg/config/config_test.go

```diff
@@ -94,11 +94,4 @@ attribution:
 		assert.Equal(t, []string{"nick@nickyt.co", "nick@opensauced.pizza"}, config.Attributions["nickytonline"])
 		assert.Equal(t, []string{"coding@zeu.dev"}, config.Attributions["zeucapua"])
 	})
-
```

Comment:
Did you want to delete this instead of comment it out?

Reply:
I had commented it out and committed it in https://github.com/open-sauced/pizza-cli/pull/141 since tests would fail with this: this just gets rid of it. I played around abit with getting [`spf13/afero`'s](https://github.com/spf13/afero) abstract, in memory filesystem to work with this. But it'd be alot of tooling / boilerplate / abstraction for a test that basically just makes sure the stdlib is doing what we expect.

Still probably worthwhile in an end to end test but that'd also probably have to be in a container or something.

Anyways, worth punting on 🏈